syncd/Syncd.cpp

```diff
@@ -1004,6 +1004,28 @@ sai_status_t Syncd::processBulkCreateEntry(
         }
         break;
 
+        case SAI_OBJECT_TYPE_MY_SID_ENTRY:
+        {
+            std::vector<sai_my_sid_entry_t> entries(object_count);
+            for (uint32_t it = 0; it < object_count; it++)
```

Comment:
plesae add empty line before each for in entire PR

Reply:
Format for MY_SID_ENTRY case is similar to other objects.

Reply:
please fix

Reply:
Done